### PR TITLE
Add section on obtaining client IP address

### PR DESCRIPTION
**Pull request by NicoPlyley**
This has been a big question on Discord. Currently this is the only way I found to solve this issue. 

I understand if you do not want it added since it may not be the ideal way since the ip address is added to the env which is meant for environment variables.

### CODE REVIEW

**Comment by yusukebe** [comment]
Hi @NicoPlyley !

I think your way is good! Plus, we can add the type by using a `Bindings` generics like the following:

```ts
import type { SocketAddress } from 'bun'
import { Hono } from 'hono'

type Bindings = {
  ip: SocketAddress
}

const app = new Hono<{ Bindings: Bindings }>()

app.get('/', (c) => {
  return c.json({
    yourIp: c.env.ip
  })
})

Bun.serve({
  fetch(req, server) {
    return app.fetch(req, { ip: server.requestIP(req) })
  }
})
```

`Bindings` seems to be only for Cloudflare, but we are now using [it in Node.js adapter](https://github.com/honojs/node-server?tab=readme-ov-file#accessing-nodejs-api).

**Comment by NicoPlyley** [comment]
Great! I made some updates with the Bindings example and output of `c.env.ip`

File: getting-started/bun.md [hunk 1]
```diff
@@ -122,3 +122,51 @@ Then, run the command.
...
 ```
+## Obtaining Request IP Address
+
+To obtain the IP address of the client, you need to use `Bun.serve`.
+
+```ts
+Bun.serve({
+  fetch(req, server) {
+    return app.fetch(req, { ip: server.requestIP(req)})
+
```
[the remark this answers is not in the document]
Comment (NicoPlyley) [reply getting-started/bun.md]: How about this, I simplified it into once example

[the remark this answers is not in the document]
Comment (yusukebe) [reply getting-started/bun.md]: Great!

**Review by yusukebe — APPROVED** [review]
LGTM!

**Comment by yusukebe** [comment]
@NicoPlyley 

Thanks Nico! Let's go with this.

**Pull request merged**